models/solution/solution.rb

```diff
@@ -96,6 +96,14 @@ def +(other)
       solution
     end
 
+    def <(other)
+      self_metrics = [self.unassigned_stops.size, self.cost]
+
+      other_metrics = [other.unassigned_stops.size, other.cost]
+
+      self_metrics <=> other_metrics
+    end
+
```

Comment:
exclusion cost is currently part of cost if it is defined for the unassigned nodes. So it is indeed not relevant to only rely on the number of unassigned stops in these cases. Moreover it would depend on priority as well. 

I will provide another formula to compare two solutions :+1: